modules/end-user-guide/pages/troubleshooting-webview-loading-error.adoc

```diff
@@ -0,0 +1,42 @@
+:_content-type: PROCEDURE
+:description: Troubleshooting Webview loading error
+:keywords: user-guide, troubleshooting-webview-loading-error
+:navtitle: Troubleshooting Webview loading error
+:page-aliases: .:troubleshooting-webview-loading-error.adoc
+
+[id="troubleshooting-webview-loading-error"]
+
+= Troubleshooting Webview loading error
```

Comment:
Looks like it's lower-case in sentences and sentence capitalization:
https://code.visualstudio.com/api/ux-guidelines/webviews
https://code.visualstudio.com/api/extension-guides/webview

Reply:
+1

Reply:
it is but until Vale says otherwise atm. I'll have to update the error rules first.